data/maps/TranquilRoute_CutHouse/scripts.pory

```diff
@@ -152,4 +152,47 @@ script TranquilRoute_CutHouse_EventScript_RandomCryGuy {
 	setflag(FLAG_MON_CRY_NPC)
 	release
 	end
-}
\ No newline at end of file
+}
+
+
+
+script TranquilRoute_CutHouse_EventScript_RandomDexGuy {
+	lock
+	faceplayer
+	if(flag(FLAG_DEX_RIDDLE_NPC_GIVE_ITEM))
+	{
+		msgbox("you already got it right\ncome back tomorrow dingus")
+		release
+		end
+	}
+
+	if(flag(FLAG_DEX_RIDDLE_NPC))
+	{
+		msgbox("you guessed wrong\ncome back tomorrow dingus")
+		release
+		end
+	}
```

Comment:
im probably not gonna do this one because i imagine the majority of this script is gonna be reworked when it gets implemented for real anyway, this is mostly just a dummy script. however @purrfectdoodle should see this comment for when it actually comes time to implement this script